libs/src/api/account.js

```diff
@@ -101,6 +101,8 @@ class Account extends Base {
           if (!hasWallet) {
             phase = phases.HEDGEHOG_SIGNUP
             const ownerWallet = await this.hedgehog.signUp(email, password)
+            // TODO: EMAIL THIS LINK
+            const recoveryLink = await this.generateRecoveryLink(email)
```

Comment:
@dmanjunath how do you think we should be handling this? i'm not sure this is ready to check in until the flow is confirmed, even if everything here is working....

Reply:
@hareeshnagaraj Agreed. Until we have the identity service actually sending the email and we can verify, we shouldn't merge it. Didn't know if you were gonna do that in a separate PR or in this one so i just approve the libs/hedgehog side so far